sample-apps/python-manual-instrumentation-sample-app/random_metrics.py

```diff
@@ -0,0 +1,93 @@
+import random
+import threading
+import time
+
+from config import *
+from opentelemetry.metrics import CallbackOptions, Observation
+from opentelemetry import metrics
+
+cfg = create_config('config.yaml')
+
+meter = metrics.get_meter(__name__)
+
+# Starts the callback for cpu usage
+def cpu_usage_callback(options: CallbackOptions):
+    min = 0
+    max = cfg['RandomCpuUsageUpperBound']
+    cpu_usage = Observation(value=random.randint(min, max))
+    print('CPU Usage asked by SDK')
+    yield cpu_usage
+
+# Starts the callback for heap size
+def heap_size_callback(options: CallbackOptions):
+    min = 0
+    max = cfg['RandomTotalHeapSizeUpperBound']
+    total_heap_size = Observation(value=random.randint(min, max))
+    print("Heapsize asked by SDK")
+    yield total_heap_size
+
+# This is the random metric collector class
+class RandomMetricCollector():
```

Comment:
nit: Should this be called `RandomMetricsProducer` or something like that so that people don't get confused with the otel-collector?

Reply:
On the other sample apps this object is called the RandomMetricCollector. It is possible to change this but the other sample apps must be changed as well. Thoughts?

Reply:
I think `Producer` is probably a better descriptor, but I'd change this and the rest of the applications in a separate change.